tables/attributeset.py

```diff
@@ -690,14 +694,5 @@ class NotLoggedAttributeSet(AttributeSet):
     def _g_log_add(self, name):
         pass
 
-
     def _g_del_and_log(self, name):
         self._g__delattr(name)
-
-
-## Local Variables:
-## mode: python
-## py-indent-offset: 4
-## tab-width: 4
-## fill-column: 72
-## End:
```

Comment:
I have removed them because `flake8` does not like double `##` as comment. But actually I believe they do not really belong to the project, since any random editor/IDE could leave some traces that would render the source code confusing.

Another thing we could consider removing is the long header of *some* files (with the name of the author(s), some date around 2003, license, etc.). Do we need these legally? There's a license with the source code, and there's the whole git history that tells much more about the author(s) and changes).

And maybe you could also just grep and hopefully delete all commented out code in the repository. I had to touch quite a lot of it in this PR without being sure whether this would be even needed. You remember much more about the code than I do.

Reply:
Please feel free to remove all the legacy emacs metainfo.

Also, I am fine with removing the copyright headers. Using a reference to a general LICENSE file (conveniently updated to include new maintainers :-) should be enough.

Reply:
Regarding the header I don't think we are forced to add the copyright statement in each file )but I could be wrong).
In any case before removing it IMHO it is better to ask to the person mentioned in the header itself.

Regarding commented code probably most of it is obsolete but in many cases there could be something useful for debugging/future enhancements.
IMHO a case by case review is necessary.

Reply:
ups, sorry @FrancescAlted I have not seen you comment before posting mine